csemonitor.go

```diff
@@ -104,6 +116,25 @@ func (reporter *Reporter) Run() {
 	}
 }
 
+func (reporter *Reporter) getServiceID() {
```

Comment:
don not interact with sc for twice just get service id directly

Reply:
The interaction will happen only when service ID deletes...
But as per your comment if we change this way we need to interact with SC every time even if service ID is correct.

Reply:
you can put this logic only in go-chassis, not in this repo

Reply:
Done